### ISSUE TITLE
Steam Machine sound is not looping constantly.

### ISSUE THREAD

**Issue by Xiarno** [GTNewHorizons/GT-New-Horizons-Modpack#17951]
### Your GTNH Discord Username

Xiarno

### Your Pack Version

2.7.0

### Your Server

Private

### Java Version

Java 21

### Type of Server

None

### Your Expectation

The sound of Steam Machines is constant when a craft is in progress.

### The Reality

The sound of Steam Machines does not loop correctly and stops after some seconds to restart after a crafting cycle.

https://github.com/user-attachments/assets/f5786736-548a-404b-a8b6-7a768523c415

### Your Proposal

Make the sound constant while a machine has a craft in progress.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by Yamnasm** [comment]
The sound is triggered with the beginning of a new craft. That also means that recipes that are processed faster than their typical sound tend to "overlap".

I'm reasonably confident that sounds only being played on a new craft is intentional, but the overlapping sounds causes dissonance, which I personally don't enjoy hearing.

**Comment by Xiarno** [comment]
> The sound is triggered with the beginning of a new craft. That also means that recipes that are processed faster than their typical sound tend to "overlap".
> 
> I'm reasonably confident that sounds only being played on a new craft is intentional, but the overlapping sounds causes dissonance, which I personally don't enjoy hearing.

How about a config? Enable/disable looping on the clientside.

**Comment by jwright159** [comment]
Yeah this is honestly one of the few complains I have with GT rn. It's not easy to hear at a glance which of my machines are active when i'm wandering around because instead of being *finished*, the recipies might just take a long time. Combine that with macerating quick stuff like clay just DEAFENS the entire local populace

**Comment by jwright159** [comment]
Wait, this has been completed? I'm on the 2.8.0 beta 2 and it's still an issue

**Comment by PlayfulPiano** [comment]
Was fixed in https://github.com/GTNewHorizons/GT5-Unofficial/pull/4718 due to backend mechanic changes.
